contracts/smart-contract-wallet/SmartAccount.sol

```diff
@@ -271,16 +271,17 @@ contract SmartAccount is
     /**
      * @dev Gnosis style transaction with optional repay in native tokens OR ERC20
      * @dev Allows to execute a transaction confirmed by required signature/s and then pays the account that submitted the transaction.
+     * @dev Function name optimized to have hash started with zeros to make this function calls cheaper
      * @notice The fees are always transferred, even if the user transaction fails.
      * @param _tx Smart Account transaction
      * @param refundInfo Required information for gas refunds
      * @param signatures Packed signature/s data ({bytes32 r}{bytes32 s}{uint8 v})
      */
-    function execTransaction(
+    function execTransaction_S6W(
         Transaction memory _tx,
         FeeRefund memory refundInfo,
         bytes memory signatures
-    ) external payable virtual override returns (bool success) {
+    ) public payable returns (bool success) {
```

Comment:
can we keep it virtual, that way we can override this as well.

Reply:
+